doc/README-2.0

```diff
@@ -162,7 +158,7 @@ Usage: pkitool [options...] [common-name]
 Options:
   --batch    : batch mode (default)
   --keysize  : Set keysize
-      size   : size (default=1024)
+      size   : size (default=4096)
```

Comment:
Well, that was fast:
* https://github.com/OpenVPN/easy-rsa-old/blob/master/easy-rsa/2.0/build-dh
Adding one parameter to vars should do it.